packages/terra-clinical-item-display/src/ItemComment.scss

```diff
@@ -3,10 +3,10 @@
 :local {
   .item-comment {
     @include terra-clinical-item-body;
-    color: var(--terra-clinical-item-comment-color, #75767d);
+    color: var(--terra-clinical-item-display-comment-color, #75767d);
```

Comment:
do we want another 'item' here to match the css class? or is that too redundant

```suggestion
    color: var(--terra-clinical-item-display-item-comment-color, #75767d);
```

Reply:
We probably should considering how ItemDisplay.sccs does has the full class names in the variables and we should try to be consistent.

Reply:
let's update it then

Reply:
Now updated.